pkg/helm/helm_helpers.go

```diff
@@ -574,7 +579,7 @@ type HelmRepoCredential struct {
 
 // DecorateWithSecrets will replace any vault: URIs with the secret from vault. Safe to call with a nil client (
 // no replacement will take place).
-func DecorateWithSecrets(options *InstallChartOptions, vaultClient vault.Client) (func(), error) {
+func DecorateWithSecrets(options *InstallChartOptions, vaultClient secreturl.Client) (func(), error) {
```

Comment:
nit: I would rename the `vaultClient` => `secreturlClient` for readability reasons.